From 0d436218d32002c29bde5d84a5b41abb915dc753 Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert (git)" <dgilbert@redhat.com>
Date: Thu, 16 Jan 2014 16:15:05 -0500
Subject: [PATCH 14/14] Add back -no-hpet but ignore it

Message-id: <1389888905-31880-1-git-send-email-dgilbert@redhat.com>
Patchwork-id: 56754
O-Subject: [RHEL-7.0 qemu-kvm PATCH 1/1] Add back -no-hpet but ignore it
Bugzilla: 1044742
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Jiri Denemark <jdenemar@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

BZ: 1044742
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=6886373
Upstream-status: Red Hat only

  Undoes some of bz947441, in that we keep the -no-hpet command line
option, for compatibility with libvirt since it has no way of knowing
that it was removed.

  Since hpet is disabled anyway, keeping the option does no harm.
---
 qemu-options.hx | 2 --
 vl.c            | 2 --
 2 files changed, 4 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-options.hx | 2 --
 vl.c            | 2 --
 2 files changed, 4 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index d9320b5..49b4bc2 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -1280,10 +1280,8 @@ it if your guest OS complains about ACPI problems (PC target machine
 only).
 ETEXI
 
-#if 0 /* Disabled for Red Hat Enterprise Linux */
 DEF("no-hpet", 0, QEMU_OPTION_no_hpet,
     "-no-hpet        disable HPET\n", QEMU_ARCH_I386)
-#endif
 
 DEF("acpitable", HAS_ARG, QEMU_OPTION_acpitable,
     "-acpitable [sig=str][,rev=n][,oem_id=str][,oem_table_id=str][,oem_rev=n][,asl_compiler_id=str][,asl_compiler_rev=n][,{data|file}=file1[:file2]...]\n"
diff --git a/vl.c b/vl.c
index b70d107..deb5884 100644
--- a/vl.c
+++ b/vl.c
@@ -3558,11 +3558,9 @@ int main(int argc, char **argv, char **envp)
             case QEMU_OPTION_no_acpi:
                 acpi_enabled = 0;
                 break;
-#if 0 /* Disabled for Red Hat Enterprise Linux */
             case QEMU_OPTION_no_hpet:
                 no_hpet = 1;
                 break;
-#endif
             case QEMU_OPTION_balloon:
                 if (balloon_parse(optarg) < 0) {
                     fprintf(stderr, "Unknown -balloon argument %s\n", optarg);
-- 
1.8.3.1

